app/views/step_by_step_pages/show/_actions.erb

```diff
@@ -14,7 +14,9 @@
         method: :post,
         class: %w(govuk-link govuk-link--no-visited-state app-link--right)
       %>
-    <% elsif @step_by_step_page.can_be_published? %>
+    <% elsif
+      @step_by_step_page.can_be_published? && !(current_user.has_permission? "Unreleased feature") || # remove this line when the feature flag is removed
+      @step_by_step_page.status.approved_2i? && (current_user.has_permission? "Unreleased feature") %>
```

Comment:
Can probably remove the `"Unreleased feature"` requirement from this line if we're capturing that on line 18 🤔

Reply:
I tried to split the conditional statement making sure we're showing the 'Publish' button for users without "Unreleased feature" permission when a page can be published and we're only showing it when the status is `approved_2i` for users without "Unreleased feature" permission. This way it's easier to remove the first line with the feature flag without re-thinking all the logic.

Reply:
Ahh I didn't spot that one was `!` and one wasn't. That makes sense now, thanks